<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailWindow</class>
 <widget class="QWidget" name="DetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DetailWindow</string>
  </property>
  <widget class="QListView" name="warAttacksList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>War Attacks:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>41</width>
     <height>431</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistics for the last 3 cycles(recent first):</string>
   </property>
  </widget>
  <widget class="QLabel" name="donations">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Donations:</string>
   </property>
  </widget>
  <widget class="QLabel" name="requests">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Requests:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ratio">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ratios:</string>
   </property>
  </widget>
  <widget class="QLabel" name="contribution">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contributions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="avgWarScore">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avg. war score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="avgWarStars">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avg. war stars:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cgScore">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CG scores:</string>
   </property>
  </widget>
  <widget class="QLabel" name="avgCGScore">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avg. CG score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="activity">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activities:</string>
   </property>
  </widget>
  <widget class="QLabel" name="role">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Role:</string>
   </property>
  </widget>
  <widget class="QLabel" name="specialRole">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Special role:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="donationsLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="requestsLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ratiosLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="contributionsLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="avgWarScoreLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="avgWarStarsLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="cgScores">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="avgCgScoreLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="activitiesLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="roleLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="specialRoleLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notes">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
